.circleci/config.yml

```diff
@@ -324,26 +349,15 @@ workflows:
       - py3_compat
       - unittests_py27
       - unittests_py36
-      - wagon:
```

Comment:
that means we test the examples with the plugin in the bundle. not the one that is on the repo(in case we changed something and didn't bump the version)